Update credit.md with ORCID cv (#1700)
alan-turing-institute/the-turing-way: book/website/reproducible-research/credit.md

- All your outputs will remain linked to your unique identifier even if you change your name or your institute. This way, you only have to enter the information once.
- To access or sign up to services that utilise ORCID, for example publisher requirements, funding management portals ([ResearchFish](ResearchFish)), the [CRIS (current research information system)](https://en.wikipedia.org/wiki/Current_research_information_system) system at your institute, like the [Zenodo](https://zenodo.org) repository.
- You can add your ORCID to your CV/resume so that anyone can have a look at all your research outputs.

> - You can also use your ORCID to create a CV using, for example, the [ROpenSci package rorcid](https://discuss.ropensci.org/t/using-rorcid-to-generate-a-website-cv/1806). 

Reviewer: ```suggestion
- You can also use your ORCID profile to create a CV using [ROpenSci package rorcid](rorcid: https://github.com/ropensci/rorcid) as described [here](https://discuss.ropensci.org/t/using-rorcid-to-generate-a-website-cv/1806). 
```
